packages/permissions/src/components/authorized/authorized.spec.tsx

```diff
@@ -283,8 +283,8 @@ describe('rendering', () => {
             });
             shallow(<Authorized {...props} />);
           });
-          it('should indicate as not authorized', () => {
-            expect(props.render).toHaveBeenCalledWith(false);
+          it('should indicate as authorized', () => {
+            expect(props.render).toHaveBeenCalledWith(true);
```

Comment:
this test case is exactly what was wrong about our logic, 
if you look at the actual and demanded permissions: 
```
demandedPermissions: ['ViewOrders'],
actualPermissions: {	        
  canViewOrders: true,	           
  canManageOrders: false,	               
},
```
the user IS authorized even if he did not have a manage orders DataFence, because in the code we also demanded the general permission `ViewOrders` and the user meets this permission.